metricflow/protocols/query_parameter.py

```diff
@@ -45,11 +45,15 @@ def date_part(self) -> Optional[DatePart]:
         raise NotImplementedError
 
 
+GroupByParameter = Union[DimensionOrEntityQueryParameter, TimeDimensionQueryParameter]
+InputOrderByParameter = Union[MetricQueryParameter, GroupByParameter]
```

Comment:
Non-blocking: should these both have the `Input` prefix? Or neither?

Reply:
Basically I just wanted to differentiate that `InputOrderByParameter` is a nested input to the `OrderByQueryParameter`. So I followed the naming convention we use for `InputMetrics` on derived metrics, if that makes sense! But I don't think that applies to the `GroupByParameter` since there's no nesting.

Reply:
Got it, thanks!